src/proposals/Addresses.sol

```diff
@@ -607,6 +630,18 @@ contract Addresses is Test, ChainIds {
             8453,
             0x627Fe393Bc6EdDA28e99AE648fD6fF362514304b
         );
+
+        /// Moonwell Rocket Pool Ether
+        _addAddress(
+            "JUMP_RATE_IRM_MOONWELL_rETH",
+            8453,
+            0x36BD01e31834A1ea271F0A135e92517a6320B511
+        );
+        _addAddress(
+            "MOONWELL_rETH",
+            8453,
+            0xCB1DaCd30638ae38F2B94eA64F066045B7D45f44
```

Comment:
Confirmed this contract is deployed and points to the correct underlying rETH contract.